libs/designer/src/lib/core/mcp/utils/serializer.ts

```diff
@@ -180,6 +183,10 @@ const generateInputsSchema = (inputs: SerializedParameter[]): any => {
   };
 };
 
+const getWorkflowNameFromOperation = (operationSummary: string | undefined, operationId: string): string => {
+  return operationSummary ? operationSummary.replaceAll(' ', '_') : operationId;
```

Comment:
nit: can use transformOperationTitle

Reply:
Created this method so in future when we are using non empty LAs we shoud be able to get unique name from a list [mostly async then], so we are actually getting workflow name not just transforming the operation title